config/shakapacker.yml

```diff
@@ -69,7 +69,9 @@ development:
     # live_reload: true
     client:
       # Should we show a full-screen overlay in the browser when there are compiler errors or warnings?
-      overlay: true
+      overlay:
+        warnings: false
+        errors: true
```

Comment:
I would prefer if you could revert this change (and keep it local). 🙏

Reply:
Sorry, I missed that.